src/testcases/ClassesTest.java

```diff
@@ -1,5 +1,383 @@
 package testcases;
 
+//import junit methods
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import org.junit.jupiter.api.Test;
+
+//import local classes to test
+import datastructures.Classes;
+import datastructures.Entity;
+import datastructures.Relationship;
+
+/** Run tests on Classes Data Structure
+ *
+ * @author Stefan Gligorevic
+ */
 public class ClassesTest {
 
+    /** create an instance of the Classes data structure
+     *
+     * assert that the relationships and entities lists are empty
+     */
+    @Test
+    public void initClass()
+    {
+        Classes initClass = new Classes();
+        assertTrue("Relationships list is empty", initClass.getRelationships().isEmpty());
+        assertTrue("Entities list is empty", initClass.getEntities().isEmpty());
+    }
+
+    /* ------------------------------------------------------------------------------ */
+    /*                          TEST HELPER/MEMBER FUNCTIONS                          */
+    /* ------------------------------------------------------------------------------ */
+
+    /** test the clear() method
+     *
+     * create classes and relationships then call clear
+     * assert that relationships and entities lists are empty after calling clear
+     */
+    @Test
+    public void testClear()
```

Comment:
We could, but I'm not sure how to check if they were deleted because the entities would be deleted with clear(), meaning I can't access any of the attributes to call .getAttributes() on them.

Reply:
Yeah I guess you're right.  I was just being really picky with this one.  I'm just gonna resolve this.